pkg/helm/actions/install_chart_test.go

```diff
@@ -1,33 +1,39 @@
 package actions
 
 import (
-       "github.com/stretchr/testify/assert"
-       "helm.sh/helm/v3/pkg/action"
-       "helm.sh/helm/v3/pkg/chartutil"
-       kubefake "helm.sh/helm/v3/pkg/kube/fake"
-       "helm.sh/helm/v3/pkg/storage"
-       "helm.sh/helm/v3/pkg/storage/driver"
-       "io/ioutil"
-       "testing"
+	"github.com/stretchr/testify/assert"
+	"helm.sh/helm/v3/pkg/action"
+	"helm.sh/helm/v3/pkg/chartutil"
+	kubefake "helm.sh/helm/v3/pkg/kube/fake"
+	"helm.sh/helm/v3/pkg/release"
+	"helm.sh/helm/v3/pkg/storage"
+	"helm.sh/helm/v3/pkg/storage/driver"
+	"io/ioutil"
+	"testing"
 )
 
 func TestInstallChart(t *testing.T) {
-	       chart := "../testdata/influxdb-3.0.2.tgz"
-	       store := storage.Init(driver.NewMemory())
-	       actionConfig := &action.Configuration{
-		               Releases:     store,
-		               KubeClient:   &kubefake.PrintingKubeClient{Out: ioutil.Discard},
-		               Capabilities: chartutil.DefaultCapabilities,
-		               Log:          func(format string, v ...interface{}) {},
-		       }
-		       act := NewHelmActions(actionConfig)
-	       _, err := act.InstallChart("test", "test", chart, nil)
-	       if err != nil {
-		               t.Error(err)
-		       }
-	       rels, err := act.ListReleases()
-	       if err != nil {
-		               t.Error(err)
-		       }
-	       assert.Len(t, rels, 1, "Release list should contain installed chart release")
-}
\ No newline at end of file
+	chart := "../testdata/influxdb-3.0.2.tgz"
+	store := storage.Init(driver.NewMemory())
+	actionConfig := &action.Configuration{
+		Releases:     store,
+		KubeClient:   &kubefake.PrintingKubeClient{Out: ioutil.Discard},
+		Capabilities: chartutil.DefaultCapabilities,
+		Log:          func(format string, v ...interface{}) {},
+	}
+	act := NewHelmActions(actionConfig)
+	_, err := act.InstallChart("test-namespace", "test", chart, nil)
```

Comment:
you should use the value returned here for assertions, not invoking `ListRelease()`. Otherwise, we are not able to catch contract breaks within `InstallChart`.

Reply:
Done